canopy_article.ml

```diff
@@ -49,14 +49,28 @@ let to_tyxml_listing_entry article =
     | Some abstract -> [p ~a:[a_class ["list-group-item-text abstract"]] [pcdata abstract]] in
   let created = ptime_to_pretty_date article.created in
   let content = [
-      h4 ~a:[a_class ["list-group-item-heading"]] [pcdata article.title];
-      span ~a:[a_class ["author"]] [pcdata author];
-      pcdata " ";
-      pcdata "("; time [pcdata created]; pcdata ")";
-      br ();
-    ] in
+    h4 ~a:[a_class ["list-group-item-heading"]] [pcdata article.title];
+    span ~a:[a_class ["author"]] [pcdata author];
+    pcdata " ";
+    pcdata "("; time [pcdata created]; pcdata ")";
+    br ();
+  ] in
   a ~a:[a_href article.uri; a_class ["list-group-item"]] (content ++ abstract)
 
+let to_tyxml_tags tags =
```

Comment:
A function to format the list of tags as links to related articles, as HTML. This is used to generates the page for the `/tags` URI